src/d3d9/d3d9_spec_constants.h

```diff
@@ -62,7 +63,8 @@ namespace dxvk {
       { 3, 0,  16 }, // VertexShaderBools
       { 3, 16, 16 }, // PixelShaderBools
 
-      { 4, 0,  16 }, // Fetch4
+      { 4, 0,  20 }, // DrefClamp
```

Comment:
Actually, just put this in the same as FETCH4.

Its not possible to do projections in vs, so it can fit.